kotlin-result/src/jsMain/kotlin/com/github/michaelbull/result/BindException.kt

```diff
@@ -0,0 +1,3 @@
+package com.github.michaelbull.result
+
+internal actual object BindException : Exception()
```

Comment:
sucks that we have to do this, if you haven't already can you upvote: https://youtrack.jetbrains.com/issue/KT-20427 ?

Reply:
Of course, that's a great suggestion!